services/core-api/app/config.py

```diff
@@ -291,9 +291,13 @@ def JWT_ROLE_CALLBACK(jwt_dict):
     CRED_DEF_ID_MINES_ACT_PERMIT = os.environ.get("CRED_DEF_ID_MINES_ACT_PERMIT",
                                                   "CRED_DEF_ID_MINES_ACT_PERMIT")
 
-    #The key pair in this did web MUST match the keypair of the did:indy:candy in Traction.
-    CHIEF_PERMITTING_OFFICER_DID_WEB = os.environ.get("CHIEF_PERMITTING_OFFICER_DID_WEB",
-                                                      "CHIEF_PERMITTING_OFFICER_DID_WEB")
+    CHIEF_PERMITTING_OFFICER_DID_WEB_VERIFICATION_METHOD = os.environ.get(
+        "CHIEF_PERMITTING_OFFICER_DID_WEB_VERIFICATION_METHOD",
+        "CHIEF_PERMITTING_OFFICER_DID_WEB_VERIFICATION_METHOD")
```

Comment:
why two lines here?

Reply:
2nd params is the default if not provided. 

App won't start if the second param is not provided. 

slightly more developer friendly this way, .env-example has the appropriate values.